src/resources/views/corporation/ledger/reprocessingfeebymonth.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('web::corporation.ledger.layouts.view', ['sub_viewname' => 'bountyprizesbymonth'])
+
+@section('title', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+@section('page_header', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
```

Comment:
This one have also to be updated. It will appear in SeAT as page title (just bellow the top ribbon).
It must match with the upper one.